SConstruct

```diff
@@ -26,7 +26,8 @@ opts.Update(env)
 # - CPPDEFINES are for pre-processor defines
 # - LINKFLAGS are for linking flags
 
-env.Append(CCFLAGS="-fdiagnostics-color")
+if ["platform"] == "linux":
```

Comment:
You need to check the host os, not the target platform. You can perfectly cross-compile on linux for windows with this compile flag.
See [`os.name`](https://docs.python.org/3/library/os.html#os.name).